### DISCUSSION TITLE
Delet mp4 from other source: redownloaded ?

### DISCUSSION THREAD

**Discussion by jualfoar** [Tzahi12345/YoutubeDL-Material#503] [Q&A]
Hi, my plex server can not play AV1 codec. 
So my question is: what happens if I delete it from plex?
Would it by redownloaded if I have a subcrition to that playlist? 
I already downgraded the quality for the downloads.

Thank you.

**Comment by GlassedSilver** [comment]
Afaik Plex deletes the file along with the entry in the DB.

if you want to avoid that what I do in Jellyfin (which should work similarly enough) I'd suggest moving or deleting manually on the file system level and cleaning the trash in Plex if you want to start anew in Plex then.

What you can also do is to store both a playable version as well as the AV1 in Plex, once Plex supports AV1 you have the better encoded file handy. As far as sharing a YTDL-M directory with Plex directly I can't really help there since I left Plex for Jellyfin myself and use YTDL-M as my exclusive frontend for my YT/Twitch/..  downloads.
